tap_hubspot/__init__.py

```diff
@@ -916,6 +920,55 @@ def sync_engagements(STATE, ctx):
     singer.write_state(STATE)
     return STATE
 
+# TODO CREATE Meetings sync
+def sync_meetings(STATE, ctx):
+    catalog = ctx.get_catalog_from_id(singer.get_currently_syncing(STATE))
+    mdata = metadata.to_map(catalog.get('metadata'))
+    schema = load_schema("meetings")
+    bookmark_key = 'updatedAt'
+    singer.write_schema("meetings", schema, ["id"], [bookmark_key], catalog.get('stream_alias'))
+    start = get_start(STATE, "meetings", bookmark_key)
+
+    current_sync_start = get_current_sync_start(STATE, "meetings") or utils.now()
+    STATE = write_current_sync_start(STATE, "meetings", current_sync_start)
+    singer.write_state(STATE)
+
+    max_bk_value = start
+    LOGGER.info("sync_meetings from %s", start)
+    params = {
+        'limit': 100,
+        'properties': "hs_internal_meeting_notes,hs_lastmodifieddate,hs_meeting_body,hs_meeting_end_time,hs_meeting_external_url,hs_meeting_location,hs_meeting_outcome,hs_meeting_start_time,hs_meeting_title,hs_timestamp,hubspot_owner_id"
```

Comment:
I know we use a lot of custom properties on Hubspot. Do those apply to this endpoint? I believe the tap somewhere fetches all custom properties and then uses them in queries like this one 🤔

Reply:
Any answer on this @SBurwash ? 🙂 I see you are now fetching the properties using the v3 endpoint which is great, but why is this list here still hard-coded?